doc/en/api/1.0.0/styles.yaml

```diff
@@ -660,33 +640,81 @@ paths:
 definitions:
 
   StyleInfoPost:
-    title: style
-    type: object
+    name: styleBody
     description: >
-    
-      The style body of a request.
-      JSON or XML style uploads follow the schema below.
+
+      The style body of a request
+      JSON or XML style uploads follow this schema. For example:
+
+        - application/xml: 
+          
+          ```
+          <style><name>roads_style</name><filename>roads.sld</filename></style>
+          ```
+
+        - application/json: 
+        
+            ```
+            {"style": { "name": "roads_style", "filename": "roads.sld"}}
+            ```
+
       Otherwise, the style body is an actual style:
 
         - application/zip: 
         
             ```
-            {ZIP file containing SLD and icons}
+            <ZIP file containing SLD and icons>
             ```
             
         - application/vnd.ogc.sld+xml:
 
-          ![](img/sld-post.png)
-
-    properties:
-      name:
-        type: string
-        description: Name of style
-        example: roads_style
-      filename:
-        type: string
-        description: Name of filename containing SLD code
-        example: roads.sld
+          ```
```

Comment:
do we need an sld 1.1 example also?

Reply:
That would probably be useful, I suppose we could generate one using the REST API.